internal/datasource/config/nginx_config_parser.go

```diff
@@ -344,7 +350,7 @@ func (ncp *NginxConfigParser) parseIncludeDirective(
 }
 
 func (ncp *NginxConfigParser) addAccessLog(accessLog *model.AccessLog,
-	accessLogs []*model.AccessLog,
+	accessLogs []*model.AccessLog, maxAccessLogFiles int,
```

Comment:
you can remove the extra argument here since you can access the maxAccessLogFiles from the ncp.agentConfig.MaxAccessLogFiles